install_config/redeploying_certificates.adoc

```diff
@@ -11,63 +11,202 @@
+
+To redeploy master, etcd, and node certificates _including_ a newly-generated or
+custom CA:
 
-====
+. If you want to use a custom CA, set the following variable in your inventory
+file:
++
 ----
 # Configure custom ca certificate
 # NOTE: CA certificate will not be replaced with existing clusters.
 # This option may only be specified when creating a new cluster or
 # when redeploying cluster certificates with the redeploy-certificates
 # playbook.
-#openshift_master_ca_certificate={'certfile': '/path/to/ca.crt', 'keyfile': '/path/to/ca.key'}
+openshift_master_ca_certificate={'certfile': '</path/to/ca.crt>', 'keyfile': '</path/to/ca.key>'}
 ----
-====
++
+If you do not set the above, then the current CA will be regenerated in the next
+step.
 
-All pods using service accounts to communicate with the {product-title} API must
-be redeployed when the {product-title} CA is replaced so the certificate
-redeploy playbook will serially evacuate all nodes in the cluster when this
-variable is set.
+. Run the *_redeploy-certificates.yml_* playbook with the
+`openshift_certificates_redeploy_ca=true` variable, specifying your inventory
+file:
++
+----
+$ ansible-playbook -i <inventory_file> \
+    playbooks/byo/openshift-cluster/redeploy-certificates.yml \
+    --extra-vars "openshift_certificates_redeploy_ca=true"
+----
+
+. As discussed previously, the playbook now serially evacuates all nodes in the
+cluster, requiring manual redeployment of registry and router certificates. See
+xref:redeploying-registry-and-router-certificates[Redeploying Registry and
+Router Certificates] to complete this procedure.
+
+[[redeploying-registry-and-router-certificates]]
+== Registry and Router Certificates
+
+When nodes are evacuated due to a redeployed CA, registry and router pods are
+restarted. This can cause outages because they cannot reach the masters using
+their old certificates.
+
+To address this issue, or to redeploy these certificates for any reason, you can
+manually redeploy registry and router certificates.
+
+[[redeploying-registry-certificates-manually]]
+=== Redeploying Registry Certificates Manually
+
+To redeploy registry certificates manually, you must add new registry
+certificates to a secret named `registry-certificates`, then redeploy the
+registry:
+
+. Set the following environment variables locally to make later commands less
+complex:
++
+----
+$ REGISTRY_IP=`oc get -n default service docker-registry -o jsonpath='{.spec.clusterIP}'`
+$ REGISTRY_HOSTNAME=`oc get -n default route/docker-registry -o jsonpath='{.spec.host}'`
+----
+
+. Create new registry certificates:
++
+----
+$ oc adm ca create-server-cert \
+  --signer-cert=/etc/origin/master/ca.crt \
+  --signer-key=/etc/origin/master/ca.key \
+  --hostnames=$REGISTRY_IP,docker-registry.default.svc.cluster.local,$REGISTRY_HOSTNAME \
+  --cert=/etc/origin/master/registry.crt \
+  --key=/etc/origin/master/registry.key \
+  --signer-serial=/etc/origin/master/ca.serial.txt
```

Comment:
Since this PR is trying to get into 3.2, 3.3, and 3.4 docs but https://github.com/openshift/openshift-docs/pull/3267 is a 3.5 feature, I'll do so in a follow-up.